cosmos-db/mongo-init.js

```diff
@@ -0,0 +1,2021 @@
+  hygieneAndStandards: {
+    code: 8015,
+    local_council: "Fakes cardiff council",
+    local_council_notify_emails: ["fsatestemail.valid@gmail.com"],
+    local_council_email: "fsatestemail.valid@gmail.com",
+    local_council_phone_number: "0300 123 6696",
+  },
+  status: {
+    registration: {
+      time: "4/2/2020, 11:34:13",
+      complete: true,
+    },
+    notifications: [
+      {
+        time: "11/12/2019, 12:37:28",
+        sent: false,
+        type: "LC",
+        address: "fsatestemail.valid@gmail.com",
+      },
+      {
+        time: "11/12/2019, 12:37:28",
+        sent: true,
+        type: "FBO",
+        address: "fsatestemail.valid@gmail.com",
+      },
+      {
+        time: "11/12/2019, 12:37:28",
+        sent: true,
+        type: "FBO_FB",
+        address: "fsatestemail.valid@gmail.com",
+      },
+      {
+        time: "11/12/2019, 12:37:28",
+        sent: true,
+        type: "FD_FB",
+        address: "fsatestemail.valid@gmail.com",
+      },
+    ],
+    tascomi: {
+      time: "11/12/2019, 12:32:33",
+      complete: true,
+    },
+  },
+  hygiene_council_code: 4221,
+  local_council_url: "cardiff",
+  source_council_id: 4221,
+  registration_data_version: "2.0.0",
+});
+db.registrations.insertOne({
+  _id: ObjectId("5dcaa838a24e940041d684bf"),
+  "fsa-rn": "PT07JA-VTZ577-ZDMAVZ",
+  collected: true,
+  collected_at: new Date("2021-01-01"),
+  createdAt: new Date("2021-01-01"),
+  updatedAt: new Date("2021-01-01"),
+  reg_submission_date: new Date("2021-01-01"),
+  establishment: {
+    establishment_details: {
+      establishment_trading_name: "Partners Co",
+      establishment_primary_number: "01111111111111",
+      establishment_secondary_number: "",
+      establishment_email: "fsatestemail.valid@gmail.com",
+      establishment_opening_date: "2019-12-22",
+    },
+    operator: {
+      operator_first_name: "Test",
+      operator_last_name: "Trader",
+      operator_postcode: "DN10 6XB",
+      operator_address_line_1: "Carrilon UK",
+      operator_primary_number: "01111111111111",
+      operator_secondary_number: "",
+      operator_email: "fsatestemail.valid@gmail.com",
+      operator_type: "SOLETRADER",
+      operator_uprn: "10023263307",
+    },
+    premise: {
+      establishment_postcode: "DN10 6XB",
+      establishment_address_line_1: "Carrilon UK",
+      establishment_type: "MOBILE",
+      establishment_uprn: "10023263307",
+    },
+    activities: {
+      customer_type: "END_CONSUMER",
+      business_type: "017",
+      business_type_search_term: "Jam",
+      import_export_activities: "NONE",
+      water_supply: "BOTH",
+      business_other_details: "",
+      opening_day_monday: false,
+      opening_day_tuesday: true,
+      opening_day_wednesday: false,
+      opening_day_thursday: false,
+      opening_day_friday: false,
+      opening_day_saturday: false,
+      opening_day_sunday: false,
+      opening_hours_tuesday: "10:00 - 12:00",
+    },
+  },
+  metadata: {
```

Comment:
These are now stored as `declaration` instead of `metadata`?

Reply:
Updated